src/setup-ros2.ts

```diff
@@ -2,179 +2,205 @@ import * as core from "@actions/core";
 import * as exec from "@actions/exec";
 
 async function runAptGetInstall(packages: string[]): Promise<number> {
-   return exec.exec(
-        "sudo",
-        ["DEBIAN_FRONTEND=noninteractive", "RTI_NC_LICENSE_ACCEPTED=yes",
-         "apt-get", "install", "--no-install-recommends", "--quiet",
-         "--yes"].concat(packages)
-    )
+	return exec.exec(
+		"sudo",
+		[
+			"DEBIAN_FRONTEND=noninteractive",
+			"RTI_NC_LICENSE_ACCEPTED=yes",
+			"apt-get",
+			"install",
+			"--no-install-recommends",
+			"--quiet",
+			"--yes"
+		].concat(packages)
+	);
 }
 
 async function runBrew(packages: string[]): Promise<number> {
-   return exec.exec(
-        "brew", ["install"].concat(packages)
-    )
+	return exec.exec("brew", ["install"].concat(packages));
 }
 
 async function installPython3Dependencies(): Promise<number> {
-   return exec.exec(
-    "sudo",
-    ["pip3", "install", "--upgrade",
-    "argcomplete",
-    "catkin_pkg",
-    "colcon-common-extensions",
-    "colcon-lcov-result",
-    "colcon-mixin",
-    "coverage",
-    "cryptography",
-    "empy",
-    "flake8",
-    "flake8-blind-except",
-    "flake8-builtins",
-    "flake8-class-newline",
-    "flake8-comprehensions",
-    "flake8-deprecated",
-    "flake8-docstrings",
-    "flake8-import-order",
-    "flake8-quotes",
-    "ifcfg",
-    "lark-parser",
-    "mock",
-    "mypy",
-    "nose",
-    "pep8",
-    "pydocstyle",
-    "pyparsing",
-    "pytest",
-    "pytest-cov",
-    "pytest-mock",
-    "pytest-repeat",
-    "pytest-rerunfailures",
-    "pytest-runner",
-     "setuptools", "wheel"]);
+	return exec.exec("sudo", [
```

Comment:
Maybe generalize this down the line like `runAptGetInstall()`

Reply:
done - but this is due to linting, not a new change